README.md

```diff
@@ -74,13 +82,6 @@ docker build . -t beerus
 docker run -e NETWORK=<arg> -e ETH_EXECUTION_RPC=<arg> -e STARKNET_RPC=<arg> -it beerus
 ```
 
-#### Examples
```

Comment:
Why remove the examples section?

Reply:
Hm, I mistakenly believe these were obsolete. They are not.